.github/workflows/continuous-integration.yml

```diff
@@ -62,7 +64,8 @@ jobs:
         uses: shivammathur/setup-php@master
         with:
           php-version: ${{ matrix.php-version }}
-          extension-csv: "intl, json, pdo"
+          pecl: true
+          extension-csv: "intl, json, pdo, ext-mbstring, ext-gd"
```

Comment:
```suggestion
          extension-csv: "gd, intl, json, mbstring, pdo"
```

Reply:
This was the first thing I tried, will give it another spin.